pkgs/tools/networking/davix/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchgit, cmake, pkgconfig, openssl, libxml2, boost }:
+
+stdenv.mkDerivation rec {
+  name = "davix-0.4.0";
+  buildInputs = [ stdenv pkgconfig cmake openssl libxml2 boost ];
+
+  src = fetchgit {
+    url = "https://github.com/cern-it-sdc-id/davix.git";
+    rev = "9d8f400ec1882602fc18312f35d617fe94ebbd67";
+    sha256 = "074i8xa6q9sf9ggqbdws64dmw5l1y17as81inl3sj9gc8n9abqvx";
+  };
+  
+  
+  meta = with stdenv.lib; {
+    description = "Davix is a toolkit designed for file operations with 
```

Comment:
Please adhere to the conventions for descriptions described in http://nixos.org/nixpkgs/manual/#idm140737319319040.